server_admin/topics/clients/oidc/con-advanced-settings.adoc

```diff
@@ -88,3 +88,13 @@ The procedure to select the algorithm is:
 . Open the client's *Settings* tab.
 . Open *Fine Grain OpenID Connect Configuration*.
 . Select the algorithm from *ID Token Encryption Content Encryption Algorithm* pulldown menu.
```

Comment:
Is this the same as a drop-down menu? If so it should be an "item list" per IBM.

Reply:
As this is text that was not added or changed by me, I would prefer not to change it as I am not sure if the pulldown menu is a dropdown menu.